client/src/routes/(auth)/login/+server.ts

```diff
@@ -8,10 +8,10 @@ import type { Client } from '@urql/core';
 import type { AccessToken, UserError } from '$lib/graphql/schema';
 import { LoginError } from './shared';
 
-export async function createToken(
+export async function _createToken(
```

Comment:
Not sure why we need to ignore it? If you don't think we need this anymore, feel free to remove please

Reply:
because it is not possible to export a function that does not give name http requests

Reply:
Thanks for updating!